src/settings.json

```diff
@@ -1,5 +1,8 @@
 {
-  "displayName": "Marketplace",
+  "displayName": {
+    "en": "Marketplace",
+    "ru": "Маркетплейс"
+  },
```

Comment:
Is this a thing?

Reply:
https://github.com/spicetify/spicetify-cli/blob/86d5df56acd81cbdd72e3316d2821aa3045f61c8/CustomApps/lyrics-plus/manifest.json

Reply:
Huh, that's cool. Didn't realize.